components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -90,6 +92,8 @@ const BillingSubscriptions = () => {
         );
     }
 
+    const showToYearlyNudge = product?.recurring_interval === RecurringIntervals.MONTH;
```

Comment:
Cloud Free and Cloud Enterprise will still have a monthly recurring interval, so as is this will show for them too. May be best to just scope to those on monthly AND professional?

Reply:
+1 - Professional customers would be the only ones that would be able to change their subscription self-serve